src/pages/Privacy.vue

```diff
@@ -31,6 +29,18 @@
                       <li>System Administrator – staff from technology services that maintain our data servers</li>
                   </ul>
 
+                  <p>This Privacy Policy also forms part of our Terms of Use – please see section 2 below.</p>
+
+                  <h3>Legitimate Interests</h3>
+
+                  <p>Under GDPR, we have determined that the lawful basis for ARC processing Rosie users’ personal data is ‘Legitimate Interests’. Broadly speaking, Legitimate Interests means that we can process personal information of Rosie users if we have a genuine and legitimate reason and we are not harming any of their rights and interests.</p>
+
+                  <p>In practice, this means that when you provide your personal data to us for Rosie, we process your data as part of the day-to-day management the Rose Voucher Project, which is part of our legitimate business interests to carry out our charitable mission & purposes, including tackling poverty and diet related ill-health, strengthening local markets and helping to build more sustainable local food economies.</p>
```

Comment:
Do you want to encode the & as `&amp;` ?

Reply:
yep!! Will do it now :) thanks for spotting